lib/spack/spack/cmd/env.py

```diff
@@ -398,6 +398,9 @@ def env_view(args):
 #
 def env_status_setup_parser(subparser):
     """print whether there is an active environment"""
+    subparser.add_argument(
+        '-x', '--exit_code', action='store_true', default=False,
+        help='Exit with nonzero status if no environment is active')
```

Comment:
I like the idea, but we should probably use a different name for the option. Pinging @becker33 and @tgamblin to see if they have something to propose. What I can think of right now is:
```suggestion
    subparser.add_argument(
        '-e', '--error', action='store_true', default=False,
        help='Exit with non-zero status if no environment is active')
```